up-core-api/uprotocol/core/udiscovery/v3/udiscovery.proto

```diff
@@ -1,5 +1,5 @@
 /*
- * SPDX-FileCopyrightText: 2023 Contributors to the Eclipse Foundation
+ * SPDX-FileCopyrightText: 2024 Contributors to the Eclipse Foundation
```

Comment:
this does not need to be changed, given that it is an existing file and unchanged major version ...